wiki/Ranking_Criteria/Code_of_Conduct/en.md

```diff
@@ -8,10 +8,10 @@ Behaviour and Conduct
 
 -   The goal of modding is to help a mapper improve their work so that it can be pushed towards Ranked status if the mapper chooses to do so. If you're not interested in helping people, steer clear of the discussion to avoid *hurting them*.
 -   When modding, consider who the post is directed at, what you aim to achieve by posting and what response you expect to receive. This will ensure your best intentions are conveyed and your post is easy to understand.
-    -   Don’t shy away from new mappers and modders - these people are the ones who can benefit the most from your input! Be patient, understanding, and provide as much explanation as possible so that they can learn and improve more quickly
-    -   If you’re new yourself, don’t be afraid to say so! This will allow more experienced community members to point you in the right direction. Everyone was new and in your position once!
--   Treat others as you wish to be treated. People are more receptive to kind, helpful suggestions than hostile ones. Try to understand others’ point-of-view rather than attack it - nobody likes to feel worthless, incompetent or stupid. If you are a masochist, assume that all others are not.
--   If you wish to comment on or discuss a beatmap, go to that beatmap’s thread - not social media outlets, or the Disqus comments section (if the map is Qualified), as it is unlikely your concerns will be heard by anyone with the power to address them appropriately.
+    -   Don't shy away from new mappers and modders - these people are the ones who can benefit the most from your input! Be patient, understanding, and provide as much explanation as possible so that they can learn and improve more quickly
+    -   If you're new yourself, don't be afraid to say so! This will allow more experienced community members to point you in the right direction. Everyone was new and in your position once!
+-   Treat others as you wish to be treated. People are more receptive to kind, helpful suggestions than hostile ones. Try to understand others' point-of-view rather than attack it - nobody likes to feel worthless, incompetent or stupid. If you are a masochist, assume that all others are not.
+-   If you wish to comment on or discuss a beatmap, go to that beatmap's thread - not social media outlets, or the Disqus comments section (if the map is Qualified), as it is unlikely your concerns will be heard by anyone with the power to address them appropriately.
 -   Personal attacks, hateful comments, misplaced joking, derailing the thread and general drama-initiation are not acceptable in any form and **will not be tolerated**.
 
 Making a Mod Post
```

Comment:
Will you be changing these headings from Setex (underline) to ATX (`#`)?

Reply:
I just changed the ones in the main RC articles since that's what I was editing-- I'll catch these with some massive regex PR eventually